src/form/Input/InlineInput/InlineInput.tsx

```diff
@@ -40,22 +50,45 @@ export interface InlineInputProps
 export const InlineInput: FC<InlineInputProps> = forwardRef(
   (
     {
+      className,
       inputClassName,
+      placeholder = 'Type here...',
       placeholderIsMinWidth = true,
       theme: customTheme,
+      extraWidth,
+      onChange,
       ...rest
     },
     ref: Ref<HTMLInputElement>
   ) => {
     const theme: InputTheme = useComponentTheme('input', customTheme);
+    const [inputValue, setInputValue] = useState('');
+    const spanStyles =
```

Comment:
Need to test this on a variety of fonts and font sizes to ensure proper spacing.

Reply:
I tested on some diff fonts and sizes, lookin good 👍

Reply:
if `font` and `text` are inherited by default, do we need to include `font-[inherit] text-[inherit]`?

Reply:
Removed font-[inherit] text-[inherit] leading-[inherit]